### PR TITLE
Add an option to invert the tray icon on Linux

### LINKED ISSUE #1780: Feture request: color of tray icon
I'm using radiance theme on ubuntu. It is bright and tray icon hard to see. Maybe it will be enough to add some shadow to current icon

### PR DESCRIPTION
**Pull request by jameshclrk**
Resolves #1780

### CODE REVIEW

File: src/main/features/core/tray.js [hunk 1]
```diff
@@ -31,21 +36,37 @@ let audioDeviceMenu = [
   },
 ];
 
-appIcon = new Tray(path.resolve(trayNormalPath, appIconFileName));
+setAppIconFileName();
+
+appIcon = new Tray(appIconFileName);
+
+Emitter.on('settings:set', (event, details) => {
```
Comment (MarshallOfSound) [inline src/main/features/core/tray.js]: This should be changed to

`Settings.onChange('appIconInvert', (newValue) => {`

File: src/main/features/core/tray.js [hunk 2]
```diff
@@ -31,21 +36,37 @@ let audioDeviceMenu = [
...
+    setAppIconFileName();
+    appIcon.setImage(appIconFileName);
+  }
+});
 
 // Change the icon if the music is playing
 Emitter.on('playback:isPlaying', () => {
-  appIcon.setImage(path.resolve(trayPlayingPath, appIconFileName));
+  currentIconPath = trayPlayingPath;
+  setAppIconFileName();
```
Comment (MarshallOfSound) [inline src/main/features/core/tray.js]: Purely to make this look nicer, why don't we make `setAppIconFileName()` return `appIconFileName` so these two lines can be simplified to.

`appIcon.setImage(setAppIconFileName())`

**Comment by MarshallOfSound** [comment]
Looking good mostly, just a couple of things to fix up 👍

**Comment by jameshclrk** [comment]
Good ideas! I'll just push the changes now.

File: src/_locales/en-US.json [hunk 3]
```diff
@@ -63,6 +63,8 @@
   "playback-label-volume-down": "Decrease Volume",
   "playback-label-volume-up": "Increase Volume",
 
+  "settings-option-invert-tray-icon": "Invert Tray Icon Color",
```
Comment (MarshallOfSound) [inline src/_locales/en-US.json]: In order to make the tests pass you have to add this key to all the other JSON files, just leave the value empty.

Comment (MarshallOfSound) [inline src/_locales/en-US.json]: > npm run test-electron fails for me on master

Heh, I'll fix those and you can rebase, gimme a sec 👍

Comment (MarshallOfSound) [inline src/_locales/en-US.json]: @jameshclrk If you rebase on master your tests should go green 👍

**Pull request merged**